scripts/suite_objects.py

```diff
@@ -157,6 +158,17 @@ def call_string(self, cldicts=None, is_func_call=False, subname=None):
                         lname = dummy
                     # end if
                 # end if
+                # Modify Scheme call_list to handle local_name change for this var.
+                # Are there any varaible transforms for this scheme?
+                # If so, change Var's local_name need to local dummy array containing
+                # transformed argument, var_trans_local.
+                if (sub_lname_list is not None) and len(sub_lname_list) > 0:
```

Comment:
Is `sub_lname_list` a somewhat ordinary list in Python? If so, then
```
if sub_lname_list:
```
is equivalent to line 165